src/Chapter1.hs

```diff
@@ -641,7 +649,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit n
+  | div n 10 == 0 = n
```

Comment:
I'm afraid, this function hangs on negative numbers because of how `div` behaves on them:

```haskell
ghci> div  (-1) 10
-1
```

Also, remember to add top-level signatures for functions!